bin/hostname/tests/hostname_test.sh

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2023 Klara, Inc.
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+#
+# These tests need to run in a multibyte locale with non-localized
+# error messages.
+#
+export LC_CTYPE=C.UTF-8
+export LC_MESSAGES=C
+
+root_dir='/usr/tests/bin/hostname'
```

Comment:
Maybe `root_dir="$(atf_get_srcdir)"`?
What if I do not run the tests under `/usr/tests`?

Reply:
Maybe even under `.`? (Then temporary directory created by kyua).

Reply:
I think `root_dir="$(atf_get_srcdir)"` is a little bit better for such case that interrupting causes the files not deleted?

Reply:
> I think `root_dir="$(atf_get_srcdir)"` is a little bit better for such case that interrupting causes the files not deleted?

That is the main reason I also suggested adding a `basic_cleanup()` with `recycle()` inside it (and not in `basic_body()`), so that the file is always removed, even when the test fails.